@navikt/core/css/button.css

```
@@ -506,6 +510,10 @@
   stroke: var(--ac-button-primary-loader-stroke-bg, rgb(255 255 255 / 0.3));
 }
 
+.navds-button:has(.navds-loader) :is(.navds-button__icon, .navds-label) {
```

Since we have the class already, i guess we can just use it here instead of `:has`
```suggestion
.navds-button--loading :is(.navds-button__icon, .navds-label) {
```

---

Could it also make sense to do this? 🤔 
```
.navds-button--loading :is(:not(.navds-loader))
```

---

loading is a state right, so maybe this should use the data attrib too? 🤔

---

As long as we add a direct child descendant, that second suggestion works as well. Can't add a second suggestion in this thread, so I'll add it locally.